js/todos.js

```diff
@@ -1,37 +1,27 @@
-var App = new (Backbone.View.extend({
+// Initialize global object
+let App = {
     Models: {},
     Views: {},
     Collections: {},
-    template: _.template(
-        '<div class="main-container">' +
-        '<div class="todo-list-container"><h1>ToDo List</h1>' +
-        '<div id="app"></div>' +
-        '</div></div>' 
-    ),
+};
 
-    render: function () {
-        this.$el.html(this.template());
-        return this;
-    },
-    start: function () {
-        this.$el.append(this.render().el);
+$(document).ready(function () {    
 
-        // Initialize some data
-        App.Todos = new App.Collections.Todos();
-        App.Todos.add({ description: 'My First Todo' });
-        App.Todos.add({ description: 'The other Todo' });
-        var initTodosView = new App.Views.TodoList({ collection: App.Todos });
-        initTodosView.render();
-        $('#app').append(initTodosView.el);
+    // Render the main container for the app
+    const appView = new App.Views.Main();
+    $(document.body).append(appView.render().el)
```

Comment:
Also, if you do that, instead of doing: `$(document.body).append(appView.render().el)` you can just tell the View which element it should be rendering into:

`const appView = new App.Views.Main({ el: '.main' });` and it'll take care of taking ownership of that div.